admin/menues.rb

```diff
@@ -96,11 +96,11 @@
 
   collection_action :load_overviewtree_as_json do
     if params[:root_id].present?
-      objects = Goldencobra::Menue.where(id: params[:root_id]).first.children.reorder(:title)
+      objects = Goldencobra::Menue.where(id: params[:root_id]).first.descendants.reorder(:title)
       cache_key ||= ["menus", params[:root_id], objects.map(&:id), objects.maximum(:updated_at)]
 
       menus = Rails.cache.fetch(cache_key) do
-        Goldencobra::Menue.find(params[:root_id]).children.order(:title).as_json(
+        Goldencobra::Menue.find(params[:root_id]).descendants.order(:title).as_json(
```

Comment:
habe ich ausprobiert und in meinen Testdaten keinen Unterschied entdeckt. Ich weiß nicht warum reorder() Haben. In menue.rb gibt es kein Default-Scope